modules/Microsoft.Network/azureFirewalls/deploy.bicep

```diff
@@ -243,6 +244,18 @@ resource azureFirewall 'Microsoft.Network/azureFirewalls@2021-08-01' = {
     applicationRuleCollections: applicationRuleCollections
     natRuleCollections: natRuleCollections
     networkRuleCollections: networkRuleCollections
+  } : {
+    firewallPolicy: empty(firewallPolicyId) ? null : {
```

Comment:
Flip the condition please. It's more intuitive to read if the value comes first, then the default

Reply:
Same for the other conditions. I see a few of those where `null` is mentioned first

Reply:
Addressed in #2547